pnpm-lock.yaml

```diff
@@ -48,7 +48,7 @@ importers:
         version: link:tools/monorepo-utils
       '@wordpress/data':
         specifier: wp-6.0
-        version: 6.6.1(react@17.0.2)
+        version: 6.6.1(react@18.3.1)
```

Comment:
Likely the reason for failing JS tests. To confirm, we can keep only changes around the deps (as per changes in package.json files):

```
 '@wordpress/env':
        specifier: ^9.7.0
```